### PR TITLE
Update device PnP samples to be aligned with DTDL sample interfaces

### PR DESCRIPTION
**Pull request by rido-min**

### CODE REVIEW

File: digitaltwins/samples/service/javascript/invoke_component_command.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').Digit
 const deviceId = process.env.IOTHUB_DEVICE_ID;
 const connectionString = process.env.IOTHUB_CONNECTION_STRING;
 const componentName = process.env.IOTHUB_COMPONENT_NAME || 'sensor'; // for the environmental sensor, try "sensor"
-const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnon'; // for the environmental sensor, you can try "blink", "turnoff" or "turnon"
+const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnOn'; // for the environmental sensor, you can try "blink", "turnoff" or "turnon"
```
Comment (ericmitt) [inline digitaltwins/samples/service/javascript/invoke_component_command.js:10]: the command in the comment should also be updated turnOff blink turnOn

Comment (rido-min) [reply digitaltwins/samples/service/javascript/invoke_component_command.js:10]: I'd rather delete de comment to avoid future maintenance

**Comment by anthonyvercolano** [comment]
>     console.log('invoking command ' + commandName + ' on component instance' + componentName + ' for device ' + deviceId + '...');

My comment to Jelani on this was that the word instance is redundant.  By definition a component is an interface instance.  So effectively one is saying "on interface instance instance".  Don't forget to leave a space after the word "component"  If you don't leave a space you'll get a log entry like "invoking command turnOn on componentsensor for device ridoDevicesAreTheBest..."  Not really sure of the purpose of the "..."

---
Refers to: digitaltwins/samples/service/javascript/invoke_component_command.js:26 in 48e0925. [](commit_id = 48e0925d5e1d6f6608a0296ce635659a567ccf8d, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
Very good attention to detail Ricardo.  Thank you for these fixes.

**Comment by anthonyvercolano** [comment]
>     this.name = new Property(true);

This MUST be fixed.

---
Refers to: digitaltwins/samples/device/typescript/src/environmentalinterface.ts:33 in 7442f6b. [](commit_id = 7442f6b8d37ac67bd47b8a6ec7c4a4db918ddbda, deletion_comment = False)

File: digitaltwins/samples/device/typescript/src/environmentalinterface.ts [hunk 2]
```diff
@@ -17,13 +17,13 @@ export class EnvironmentalSensor extends BaseInterface {
   humid: Telemetry;
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/typescript/src/environmentalinterface.ts:17]: This is humidity in the .js sample

File: digitaltwins/samples/device/typescript/src/environmentalinterface.ts [hunk 3]
```diff
@@ -17,13 +17,13 @@ export class EnvironmentalSensor extends BaseInterface {
...
+  turnOff: Command;
+  turnOn: Command;
   name: Property;
   brightness: Property;
 
   constructor(componentName: string, propertyChangedCallback?: PropertyChangedCallback, commandCallback?: CommandCallback) {
-    super(componentName, 'dtmi:my_company:com:EnvironmentalSensor;1', propertyChangedCallback, commandCallback);
+    super(componentName, 'dtmi:com:example:EnvironmentalSensor;1', propertyChangedCallback, commandCallback);
     this.temp = new Telemetry();
     this.humid = new Telemetry();
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/typescript/src/environmentalinterface.ts:28]: >  this.humid = new Telemetry(); [](start = 2, length = 31)

Need to change this also.

**Comment by anthonyvercolano** [comment]
>     await dtClient.sendTelemetry(environmentalSensor, { temp: 1 + (Math.random() * 90), humid: 1 + (Math.random() * 99) });

One should be temperature and the other should be humidity

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:93 in 48e0925. [](commit_id = 48e0925d5e1d6f6608a0296ce635659a567ccf8d, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
>     await digitalTwinClient.sendTelemetry(environmentalSensor, { temp: 1 + (Math.random() * 90), humidity: 1 + (Math.random() * 99) });

Should be temperature


---
Refers to: digitaltwins/samples/device/javascript/sample_device.js:114 in 48e0925. [](commit_id = 48e0925d5e1d6f6608a0296ce635659a567ccf8d, deletion_comment = False)

**Comment by rido-min** [comment]
> > ```
> > this.name = new Property(true);
> > ```
> 
> This MUST be fixed.
> 
> Refers to: digitaltwins/samples/device/typescript/src/environmentalinterface.ts:33 in [7442f6b](https://github.com/Azure/azure-iot-sdk-node/commit/7442f6b8d37ac67bd47b8a6ec7c4a4db918ddbda). [](commit_id = [7442f6b](https://github.com/Azure/azure-iot-sdk-node/commit/7442f6b8d37ac67bd47b8a6ec7c4a4db918ddbda), deletion_comment = False)

What needs to be fixed?

**Comment by anthonyvercolano** [comment]
Carried over with #802

**Pull request closed without merging**